src/Symfony/Bundle/FrameworkBundle/Resources/config/session.xml

```diff
@@ -37,11 +37,13 @@
 
         <service id="session.flash_bag" class="Symfony\Component\HttpFoundation\Session\Flash\FlashBag">
             <factory service="session" method="getFlashBag" />
+            <deprecated>The "%service_id%" service is deprecated since Symfony 5.1. Use $session->getFlashBag() instead</deprecated>
         </service>
         <service id="Symfony\Component\HttpFoundation\Session\Flash\FlashBagInterface" alias="session.flash_bag" />
 
         <service id="session.attribute_bag" class="Symfony\Component\HttpFoundation\Session\Attribute\AttributeBag">
             <factory service="session" method="getAttributeBag" />
+            <deprecated>The "%service_id%" service is deprecated since Symfony 5.1. Use $session->getAttributeBag() instead</deprecated>
```

Comment:
`The "%service_id%" service is deprecated since Symfony 5.1, use "$session->getAttributeBag()" instead.`